site/profile/manifests/windows/baseline.pp

```diff
@@ -8,6 +8,11 @@
     apply => immediately,
   }
 
+  service { 'wuauserv':
+    ensure => running,
+    enable => true,
```

Comment:
Need to watch out for this.  On Server 2012 Windows Update is a Manual service with a Trigger.  Whereas, enable => true will set it to Automatic.

Happy to merge but a point for later e.g. PUP-6641